packages/framework/undo-redo/src/test/sequenceHandler.spec.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under the MIT License.
  */
 
+// eslint-disable-next-line unicorn/prefer-node-protocol
```

Comment:
Same as above..

Reply:
To use "node:assert" here, I have to remove 
~~~
overrides: [
		{
			// Rules only for test files
			files: ["*.spec.ts", "src/test/**"],
			rules: {
				// Test files are run in node only so additional node libraries can be used.
				"import/no-nodejs-modules": ["error", { allow: ["assert"] }],
			},
		},
	],
~~~
in [eslintrc.cjs](https://github.com/microsoft/FluidFramework/pull/24382/files#diff-3977ca47573a55cadb62426a97db265f395fab948c9b0015d7992c6d4c715517). I think disable this eslint rule make more sense

Reply:
Ah, I think we might need to update the rule override to say `allow: ["node:assert]`. That seems to be what we do in other packages that are already using the recommended eslint config.